apps/meteor/server/methods/browseChannels.js

```diff
@@ -104,16 +104,22 @@ async function getChannelsAndGroups(user, canViewAnon, searchTerm, sort, paginat
 	};
 }
 
-const getChannelsCountForTeam = mem((teamId) => Promise.await(Rooms.findByTeamId(teamId, { projection: { _id: 1 } }).count()), {
-	maxAge: 2000,
-});
+const getChannelsCountForTeam = mem(
+	(teamId) => {
+		return Rooms.findByTeamId(teamId, { projection: { _id: 1 } });
```

Comment:
as far as i understand this returns a cursor, cursors are VERY FAST, they only  generates a object structure, no cache needed